README.md

```diff
@@ -113,9 +115,25 @@ The filename generated are based on the test that are running, ideally: `[Test-s
 
 Just as a convention.
 
+## Tui Terminal (Experimental)
+
+This is just a show case and has not to be considered following the semantic versioning of the project,
+it implies things might change with no objective reason at all.
+
+It might could be detached completely and be its own library using this one.
+
+At the moment is really very basic, error-prone and not well designed neither.
+
+There is an example too among the examples.
+
+### Screenshot
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings] Do not use headings with similar content (96:1)](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5472420)